hw/ip/prim/dv/prim_lfsr/data/prim_lfsr_cover.cfg

```diff
@@ -0,0 +1,17 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifer: Apache-2.0
+//
+// The FPV testbench should be considered the verification source of truth
+// for this module and hits 100% coverage.
+// This DV testbench exists solely to functionally test one feature of the
+// module, as a result it is safe to blindly exclude anything that is uncovered
```

Comment:
IMO remove this statement - it kinda says collecting coverage on this module is completely pointless (if that is true, then we should rather disable the coverage collection altogether).

Reply:
Hmm I think I see what you mean, thanks for bringing that up! I'll remove this line from the two cover.cfg files (and from commit message as well).